x/auth/client/testutil/suite.go

```diff
@@ -984,6 +984,10 @@ func (s *IntegrationTestSuite) TestCLIMultisign() {
 	multisigRecord, err := val1.ClientCtx.Keyring.Key("multi")
 	s.Require().NoError(err)
 
+	dummyMultisig := "cosmos1hd6fsrvnz6qkp87s3u86ludegq97agxsdkwzyh"
+	dummyMultisigAddr, err := sdk.AccAddressFromBech32(dummyMultisig)
+	s.Require().NoError(err)
```

Comment:
Let's add this in a new test, `TestSignWithMultisig`, as to not pollute the flow of the original Multisign test